ansible/main.yml

```diff
@@ -0,0 +1,110 @@
+---
+- hosts: all
+  vars:
+    app_path: /home/ubuntu/RD-WATCH
+    venv: "{{ app_path }}/venv"
+  tasks:
+  - name: checkout code
+    git:
+      repo: "https://github.com/ResonantGeoData/RD-WATCH.git"
+      dest: "{{ app_path }}"
+      force: yes
+
+  - name: Set environment variables
+    set_fact:
+      watch_environment:
+        DJANGO_CONFIGURATION:  "{{ lookup('env', 'DJANGO_CONFIGURATION') }}"
+        DJANGO_DATABASE_URL: "{{ lookup('env', 'DJANGO_DATABASE_URL') }}"
+        DJANGO_CELERY_BROKER_URL: "{{ lookup('env', 'DJANGO_CELERY_BROKER_URL') }}"
+        DJANGO_MINIO_STORAGE_ENDPOINT: "{{ lookup('env', 'DJANGO_MINIO_STORAGE_ENDPOINT') }}"
+        DJANGO_MINIO_STORAGE_ACCESS_KEY: "{{ lookup('env', 'DJANGO_MINIO_STORAGE_ACCESS_KEY') }}"
+        DJANGO_MINIO_STORAGE_SECRET_KEY: "{{ lookup('env', 'DJANGO_MINIO_STORAGE_SECRET_KEY') }}"
+        DJANGO_STORAGE_BUCKET_NAME: "{{ lookup('env', 'DJANGO_STORAGE_BUCKET_NAME') }}"
+
```

Comment:
@banesullivan We also need to know if there are credentials for Boto3 here, or whether it's done implicitly via an EC2 Instance Profile.

Reply:
It should be done implicitly via an EC2 Instance Profile